### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by jgrewe**

### CODE REVIEW

File: docs/use_cases.rst [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 2) Use of NIX as a storage backend for the `neo <http://neuralensemble.org/neo>`__ data model.
 3) Use of NIX to store EEG data.
-
+4) Use of NIX as a storage backend for `SpikeInterface <https://spikeinterface.readthedocs.io/en/latest/>`__.
 
 .. toctree::
    relacs
    neonixio
    eegbase
+   spikeinterface
```
Comment (achilleas-k) [inline docs/use_cases.rst]: I guess I'll have to write this (and the Neo one) at some point? :)

Comment (jgrewe) [reply docs/use_cases.rst]: yes, that would be most welcome ;)

**Comment by achilleas-k** [comment]
Should we merge this and continue working on the docs in subsequent PRs?

**Comment by jgrewe** [comment]
👍 added the last changes because it was still opened. Feel free to merge, if you are happy with it

**Comment by achilleas-k** [comment]
Merging since I think the codecov tests are going to stay stuck there.

**Pull request merged**